packages/strapi-plugin-graphql/services/Resolvers.js

```diff
@@ -59,7 +59,9 @@ module.exports = {
       }
 
       // Add timestamps attributes.
-      if (_.get(model, 'options.timestamps') === true) {
+      const isArrayTimestamps = _.isArray(_.get(model, 'options.timestamps'));
```

Comment:
Since I worked on that point `_.get(model, 'options.timestamps')` will be an `array` or `false`.
So we can remove this line and use `_.isArray(_.get(model, 'options.timestamps')` for the `if condition`

Reply:
options.timestamps will not be a boolean anymore ? It seems like my models got bool by default, that's why I was checking if this options was true or an array